### ISSUE TITLE
Room Switch on New-Mailbox cmdlet

### ISSUE THREAD

**Issue by dotnvo** [MicrosoftDocs/office-docs-powershell#10399]
Per [docs](https://learn.microsoft.com/powershell/module/exchange/new-mailbox#-room) the Room switch should logon-disable an account in Exchange Online, preventing users from signing into the mailbox directly. 

> When you use this switch, a logon-disabled account is created with the room mailbox, which prevents users from signing in to the mailbox. 

However, after testing in 4 different tenants, the account is 'enabled' and as long as you know the PW or can reset the PW, you can login to the account/mailbox directly. 

Example Command: `New-Mailbox -Name 'TestRoom' -Alias 'TestRoom' -DisplayName 'Test Room' -Room -ResourceCapacity 10`

![image](https://user-images.githubusercontent.com/29506123/215871165-56979608-2bfb-42ea-ab24-c70deea2114f.png)

Obviously with this command - we aren't setting a password, so we'd have to 'reset' the PW to log into the account, but from testing that works (as it probably should since the account is enabled). Only by disabling the account do we get the account lock out error, but it still lets us manage the mailbox via Full Access permissions.

Is this by design? It seems there's either a bug in the cmdlet/scope or an issue in the docs, or perhaps I'm misunderstanding 'login disabled' and it is not synonymous with 'Account enabled', and need further explanation on what logon-disabled actually means in this context.  Seeing as this could be a doc error, I wanted to start here to seek clarification and input, and perhaps the doc team can engage the Exchange team to see if what we are experiencing is correct or not.

I believe doing this in on premise exchange disables the account so no one can authenticate to the account as that account. However, you could still authenticate as a different user to configure calendar processing, look at the inbox, etc. for that mailbox using Full Access perms.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: d1aa38f7-88eb-7d2e-0ed6-efc3186eb96b
* Version Independent ID: 0dcef152-ab12-c711-742d-5d4e570ea075
* Content: [New-Mailbox (ExchangePowerShell)](https://learn.microsoft.com/en-us/powershell/module/exchange/new-mailbox?view=exchange-ps)
* Content Source: [exchange/exchange-ps/exchange/New-Mailbox.md](https://github.com/MicrosoftDocs/office-docs-powershell/blob/main/exchange/exchange-ps/exchange/New-Mailbox.md)
* Service: **exchange-powershell**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by chrisda** [comment]
@dotnvo, your observations are correct. I remember dealing with these issues a few years ago.

The statement in the Room switch description needs to be clarified that it applies to on-premises Exchange. Exchange Online behaves the way you described (an account is created with an unknown password, and that account is visible and manageable in Azure AD PowerShell or the Microsoft 365 admin center just like any 'ol account).

In Exchange Online PowerShell, I tried to create a new room mailbox with EnableRoomMailboxAccount $false, and I got the following error: `The -RoomMailboxPassword cannot be set for this room if -EnableRoomMailboxAccount is false.`. 

I used EnableRoomMailboxAccount $false in Set-Mailbox on the room mailbox, yet I was still able to use the account to log in to the mailbox.

In short, I believe the EnableRoomMailboxAccount and RoomMailboxPassword parameters are meaningless in the cloud. I updated the descriptions accordingly.

I also updated the description of the Room parameter to talk about this stuff.

See the linked pull request for details.

OK?

**Comment by dotnvo** [comment]
@chrisda  Sounds good thanks, wicked fast turnaround!

**Comment by chrisda** [comment]
@dotnvo, thanks. I wonder if the live account thing in the cloud is related to how accounts work in the cloud, or if it's related to other Microsoft 365 features that need the room mailbox to have a live account. For example, the EnableRoomMailboxAccount parameter description had this text (which I edited and kept, but it was likely meant for on-prem Exchange):

> Typically, the account that's associated with a room mailbox is disabled. However, you need to enable the account for features like the Skype for Business Room System or Microsoft Teams Rooms.

If you disable accounts for room mailboxes, but then you discover something else stops working, please report back here, and I'll further update the documentation.
